test/cljs/cider/nrepl/middleware/cljs_inspect_test.clj

```diff
@@ -6,18 +6,19 @@
 
 (def nil-result ["(\"nil\" (:newline))"])
 
-(def var-result ["(\"Class\" \": \" (:value \"clojure.lang.Var\" 0) (:newline) \"Meta Information: \" (:newline) \"  \" (:value \":ns\" 1) \" = \" (:value \"clojure.core\" 2) (:newline) \"  \" (:value \":name\" 3) \" = \" (:value \"*assert*\" 4) (:newline) \"Value: \" (:value \"true\" 5))"])
+(def var-result ["(\"Class\" \": \" (:value \"clojure.lang.Cons\" 0) (:newline) \"Contents: \" (:newline) \"  \" \"0\" \". \" (:value \"var\" 1) (:newline) \"  \" \"1\" \". \" (:value \"cljs.core/*assert*\" 2) (:newline))"])
```

Comment:
When all is said and done, these fixes should be squashed (at least in groups, ie. all changes to cljs_inspect_test.clj) to make the overall commit history more sensible.